torch/csrc/jit/python/init.cpp

```diff
@@ -1735,7 +1735,7 @@ void initJITBindings(PyObject* module) {
                                        const py::kwargs& kwargs) {
                     ToIValueAllowNumbersAsTensors g(allow_numbers_as_tensors);
                     return _get_operation_for_overload_or_packet(
-                        {op}, symbol, args, kwargs, /*is_overload*/ true, dk_);
+                        op, symbol, args, kwargs, /*is_overload*/ true, dk_);
                   });
               return py::make_tuple(
                   func, func_dk, py::cast(op->getTags().vec()));
```

Comment:
These values should be std::move to prevent ref count increases

Reply:
I don't see anything movable here; if we move `op` from the lambda capture then the lambda won't work if called a second time.